testsuite/domain/src/test/resources/host-configs/host-slave.xml

```diff
@@ -141,6 +141,8 @@
         </server>
         <server name="reload-two" group="reload-test-group"  auto-start="false">
             <jvm name="default" />
+            <!-- this is for ServerRestartRequiredTestCase.testSocketBinding() unless the socket-binding-group is actually used for something it is ignored on the slaves, this is a quick fix before correcting the test -->
+            <socket-bindings socket-binding-group="reload-sockets" port-offset="1550"/>
```

Comment:
Unless I've missed something, the sbg is not unused.

server-config=reload-two points to server-group=reload-test-group which points to socket-binding-group=reload-sockets

That chain is sufficient to establish use of the sbg. (It's analogous to the chain that establishes uses of a profile.)

Adding a socket-bindings child under the server-config is solely to allow overriding of the configuration obtained from the server-group. Its presence is not necessary to establish "use" of an sbg.